src/modules/HTTPTransport/types.ts

```diff
@@ -0,0 +1,15 @@
+export enum METHODS {
+  GET = 'GET',
+  POST = 'POST',
+  PUT = 'PUT',
+  DELETE = 'DELETE',
+}
+
+export type Options = {
+  headers?:Record<string, string>
+  method: METHODS;
```

Comment:
Стоит пробежаться и привести стиль к единому
```suggestion
  headers?: Record<string, string>;
  method: METHODS;
```